pipeline/extract.py

```diff
@@ -64,28 +51,19 @@ def get_relevant_plant_data() -> list[dict]:
     list_of_plants = []
 
     for i in range(51):
```

Comment:
Chris isn't a fan of 'magic numbers' 
Maybe we could have a global / env variable called LAST_PLANT_ADDRESS, or similar

Reply:
Great point!